doc/faq.rst

```diff
@@ -100,12 +100,31 @@ See :ref:`adding_graphical_models`.
 Will you add GPU support?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-No, or at least not in the near future. The main reason is that GPU support
-will introduce many software dependencies and introduce platform specific
-issues. scikit-learn is designed to be easy to install on a wide variety of
-platforms. Outside of neural networks, GPUs don't play a large role in machine
-learning today, and much larger gains in speed can often be achieved by a
-careful choice of algorithms.
+A limited but growing :ref:`list of scikit-learn estimators
```

Comment:
```suggestion
Since 20XX, a limited but growing :ref:`list of scikit-learn estimators
```
Having a year would make clear that this is a recent change.